### ISSUE TITLE
paper-tabs should use theme colors by default

### ISSUE THREAD

**Issue by osechet** [PolymerElements/paper-tabs#114]
The default background color, text color and selection bar color should be set using the default-theme's values (as it is done with paper-toolbar): --default-primary-color, --text-primary-color, --accent-color.

**Comment by osechet** [comment]
As a side note, I would like to say, that the `paper-tab-ink` does not look good if the `--accent-color` is used. I think it is better to use the `--dark-primary-color` as it is done with buttons.

**Comment by abdonrd** [comment]
Yeah! I proposed this PR: https://github.com/PolymerElements/paper-tabs/pull/55

**Comment by osechet** [comment]
Reading at the `paper-button` documentation I noticed the ink color is set as the button's color at 25% opacity (not the dark-primary-color as i suggested earlier). The `paper-tab` should probably work the same way for consistency.

**Comment by motss** [comment]
@osechet IMHO, `.25` for `opacity` of the ripple effect on `paper-tab` seems too low. You won't be able to see the ripple easily and I think at least `.5` should be fine. What do you think?

**Comment by osechet** [comment]
I don't known. My suggestion is based on what I saw in the `paper-button` doc. If the effect is OK for the button, it should be OK or the tab.
